README.md

```diff
@@ -35,21 +35,23 @@ workaround:
 
 ## How to install *realpath_turbo*
 
-1. Download the *realpath_turbo* source code.
-
-2. Extract and compile the extension
-
-   ```sh
-   $ tar -xaf realpath_turbo*
-   $ cd realpath_turbo*
-   $ phpize
-   $ ./configure
-   $ make
-   # make install
-   ```
-
-3. Adjust your `php.ini` to load and configure *realpath_turbo* extension.
-
+```sh
+wget -qO- https://github.com/Whissi/realpath_turbo/archive/master.tar.gz | tar -xz

```

Comment:
Well, that's the problem: We don't have a *static* URL: If you do `git clone` you don't have to extract something. Yes, you can download any state from GitHub, than you have something to extract. Or you can download from GitHub's release system...

Because the README is part of the archive (not only GitHub-focused) I'd like to keep the README more general. I.e. I expect that any user who want to use this extension is able to understand how to fetch a source.